css/mobile.css

```diff
@@ -326,6 +331,79 @@ header {
   padding: 0;
 }
 
+fieldset {
+  border: none;
+  text-align: center;
```

Comment:
- [ ] [REQUIRED] Please ensure you stick to the design templates in figma as much as possible, this includes the font, colours, images and spacing (margin/padding). I think you can change this to `text-align: left;` so as to match the original design

Reply:
Done.